.eslintrc.json

```diff
@@ -2,6 +2,18 @@
   "extends": [
     "@pixi/eslint-config"
   ],
+  "parserOptions": {
+    "project": ["./tsconfig.eslint.json"]
+  },
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "rules": {
+        "@typescript-eslint/comma-dangle": [1, "always-multiline"],
+        "comma-dangle": "off"
+      }
+    }
+  ],
```

Comment:
The `@typescript-eslint/comma-dangle` rule requires the `parserOptions` for project aware rules and since the config is different per project i've added in manually here instead of in the eslint config